### ISSUE TITLE
Refactoring needed

### ISSUE THREAD

**Issue by Nichodon** [anematode/gatm#15]
This is something @anematode mentioned many months ago, but basically, the code needs to be refactored at some point if we are to make this easy for others to edit in the future, as I am sure will happen.

The main issue is that **none of the code is indented and it's really hard to read**.

(This is assigned to me because it is not content related to a dumbus like me can do it).

**Comment by anematode** [comment]
Yes, the code is terrifyingly messy in places. I think there are automatic LaTeX formatters out there. I wish LaTeX wasn't such a write-only language... maybe I'm just inexperienced but it feels that way sometimes XD

**Comment by Nichodon** [comment]
TeXstudio at least makes _some_ rudimentary steps to indent but they are inaccurate at best.

**Comment by Nichodon** [comment]
Popping in here in 2022 to say that I will never be touching asy code as long as I can prevent it for the rest of my life. Bye!
